config/settings.yml

```diff
@@ -493,6 +493,15 @@ development: &development
   <<: *defaults
   log_level: debug
   password: 'secret'
+  directors:
```

Comment:
Can you please check if boshlite2 setting(https://github.com/SAP/service-fabrik-broker/blob/master/config/settings.yml#L533) is affected for this change or needs changes?

Reply:
`director` is no longer a valid property. It must be `directors`. Since i was testing in development, i only fixed that config. Please note this should have been updated as part of the BOSH user story. I have anyways updated the same for other envs as well.

Reply:
Thanks @jagadish-kb